.github/workflows/cont_integration.yml

```diff
@@ -1,4 +1,4 @@
-name: bdk-ffi CI
+name: Rust Layer CI
```

Comment:
small nit.. but for consistency with having only the first word capitalized (CI is ok since it's an acronym):
```suggestion
name: Rust layer CI
```

Reply:
Good catch.